serverless/functions/reference-content/code-examples.mdx

```diff
@@ -102,7 +102,7 @@ def handle(event, context):
 
 With event objects you can pass information through your HTTP request. They are composed of the following objects:
 
-* `pathParameters`: map(string)string - Parameters defined in the path of the HTTP Request
+* `path`: string - Parameters defined in the path of the HTTP Request
 * `queryStringParameters`: map(string)string - Query Strings parameters of the HTTP Request
 * `body`: string|byte() - Body of the HTTP Request, you will have to parse it in your handler to use it properly.
```

Comment:
From my understanding, in the runtimes that use event/context, the body will always be a string due to the JSON deserialization we do.